src/components/navbar/navbar-desktop.tsx

```diff
@@ -3,9 +3,9 @@ import React from "react";
 
 import { SpaceShuttle } from "../icons/space-shuttle";
 import { NavLink } from "../nav-link";
-import classes from "./navbar.module.css";
+import classes from "./navbar-desktop.module.css";
 
-const Navbar: React.FC = () => {
+const NavbarDesktop: React.FC = () => {
   const { pathname } = useRouter();
   return (
     <nav className="hidden md:block mt-4">
```

Comment:
Would you mind replacing the `#` in `href` attributes in this file ? Since you put the expected path for the mobile version, we could do the same here, no ?

Reply:
Sure thing, that's a good idea.
While we are at it, do you think we also should add the under line for mobile? Like this
![image](https://user-images.githubusercontent.com/31956107/144017447-c231f554-82c8-47bb-99dd-3d88fb7a18b6.png)

Reply:
ee9cf26

Reply:
> While we are at it, do you think we also should add the under line for mobile? Like this

You mean, to the active link ?

Reply:
Yes.

Reply:
As you want.

Reply:
Done 8f0f067.